includes/html/pages/devices.inc.php

```diff
@@ -14,19 +14,18 @@
 */
 
 function show_device_group($device_group_id) {
-    $device_group_name = DB::table('device_groups')->where('id', $device_group_id)->value('name');
     ?>
     <div class="panel-heading">
         <span class="devices-font-bold">
         <?php
         if ($device_group_id == 'none') {
-            echo "ungrouped Devices";
+            echo "Ungrouped Devices";
         } elseif ($device_group_id) {
             echo "Device Group: ";
+            echo htmlentities(\App\Models\DeviceGroup::where('id', $device_group_id)->value('name'));
```

Comment:
This seems to change the behavior.
1. moves the name inside the span
2. value() can still return null.